### ISSUE TITLE
[Question] Loss suddenly increases during fine-tuning with LoRA

### ISSUE THREAD

**Issue by QAQdev** [haotian-liu/LLaVA#1301]
### Question

<img width="2190" alt="image" src="https://github.com/haotian-liu/LLaVA/assets/73159641/c95195b5-92a5-4bb9-a682-b25c613bba45">
Hi, community, I encountered a strange sudden increase of loss during fine-tuning llava-v1.5-7b with LoRA, using my custom dataset. This case does not happen when I set the epoch to 1 or 2.
I would like to know the possible reasons for this happening, thanks for help in advance.

**Comment by The-kamisato** [comment]
Do you find out why? I am faced with the same problem

**Comment by QAQdev** [comment]
> Do you find out why? I am faced with the same problem

yeah, roughly because of learning_rate, i decreased it by 10x lower

**Comment by bang123-box** [comment]
hello, when we decrease the lr, how dose the performance change?

**Comment by QAQdev** [comment]
> hello, when we decrease the lr, how dose the performance change?

Hi, I remembered that the performance increased because the loss looked much better

**Comment by HangerYang** [comment]
Did you decrease the learning rate for just the llm, or both llm and the projector? Thanks!

**Comment by euyis1019** [comment]
Can you guys share the learning rate strategy? I encounter the similar question now. Thanks a lot!!!
